fe/fe-core/src/main/java/com/starrocks/http/rest/RestBaseAction.java

```diff
@@ -110,7 +122,7 @@ public String getErrorRespWhenUnauthorized(AccessDeniedException accessDeniedExc
                 List<String> activatedRoles = authorizationMgr.getRoleNamesByRoleIds(context.getCurrentRoleIds());
                 List<String> inactivatedRoles =
                         authorizationMgr.getInactivatedRoleNamesByUser(userIdentity, activatedRoles);
-                return "Access denied for user " + userIdentity  + ". " +
+                return "Access denied for user " + userIdentity + ". " +
                         String.format(ErrorCode.ERR_ACCESS_DENIED_HINT_MSG_FORMAT, activatedRoles, inactivatedRoles);
             }
             return "Access denied.";
```

Comment:
Will it be too simple to let user know the error reason?

Reply:
this is intended to do so, because we cannot get any more specific info.